### PR TITLE
Added link to checksum and verification instructions on the Downloads page

### PR DESCRIPTION
**Pull request by rohantmp**
Signed-off-by: Rohan CJ <rohantmp@gmail.com>

### CODE REVIEW

**Comment by rohantmp** [comment]
Noticed there was no link when downloading. Resolves: issue #54

**Comment by rohantmp** [comment]
@sanjabonic

File: pages/download/index.html [hunk 1]
```diff
@@ -23,6 +23,14 @@ <h1>Download a Silverblue image</h1>
 <p>You may then install Silverblue by booting from your USB Drive. <b>Note:</b> it is
 advisable to keep the default partitioning setup that the Installer suggests,
 since rpm-ostree expects this setup.</p>
+<p> You can verify your download with 
+<a href="https://mirrors.huaweicloud.com/repository/fedora/releases/29/Silverblue/x86_64/iso/Fedora-Silverblue-29-1.2-x86_64-CHECKSUM">
```
Comment (miabbott) [inline pages/download/index.html]: This points to a specific mirror; it should instead point to the redirect URL

https://download.fedoraproject.org/pub/fedora/linux/releases/29/Silverblue/x86_64/iso/Fedora-Silverblue-29-1.2-x86_64-CHECKSUM

File: pages/download/index.html [hunk 2]
```diff
@@ -31,6 +39,15 @@ <h1>Try the latest</h1>
 
 <p><a class="button" href='https://download.fedoraproject.org/pub/fedora/linux/releases/test/30_Beta/Silverblue/x86_64/iso/Fedora-Silverblue-ostree-x86_64-30_Beta-1.8.iso'><svg class="icon"><use href="/public/svg-sprites.svg#download" /></svg>64-bit 2.1GB Silverblue image (Fedora 30 Beta)</a></p>
 
+<p> You can verify your download with 
+<a href="http://ftp.jaist.ac.jp/pub/Linux/Fedora/releases/test/30_Beta/Silverblue/x86_64/iso/Fedora-Silverblue-30_Beta-1.8-x86_64-CHECKSUM">
```
Comment (miabbott) [inline pages/download/index.html]: Same as above.

https://download.fedoraproject.org/pub/fedora/linux/releases/test/30_Beta/Silverblue/x86_64/iso/Fedora-Silverblue-30_Beta-1.8-x86_64-CHECKSUM

**Comment by sanjabonic** [comment]
@rohantmp Could you please make the changes @miabbott suggested? We can then merge the PR.

**Comment by rohantmp** [comment]
@sanjabonic done!

**Comment by pnemade** [comment]
I think commits could have been squashed into single commit

**Pull request merged**